### PR TITLE
Update release asset checker for v13

### PR DESCRIPTION
**Pull request by takagi**

### CODE REVIEW

File: check_release_assets.py [hunk 1]
```diff
@@ -22,35 +22,45 @@
...
+    '13': [
+        ('cupy-cuda102',  _v13_cuda_matrix + _v13_aarch64_matrix),
+        ('cupy-cuda110',  _v13_cuda_matrix),
+        ('cupy-cuda111',  _v13_cuda_matrix),
+        ('cupy-cuda11x',  _v13_cuda_matrix + _v13_aarch64_matrix),
+        ('cupy-cuda12x',  _v13_cuda_matrix + _v13_aarch64_matrix),
+        ('cupy-rocm-4-3', _v13_rocm_matrix),
+        ('cupy-rocm-5-0', _v13_rocm_matrix),
+    ],
+    # v11.x
```
Comment (asi1024) [inline check_release_assets.py:63]:
```suggestion
    # v12.x
```

File: check_release_assets.py [hunk 2]
```diff
@@ -22,35 +22,45 @@
 
 sdist_project = 'cupy'
 
+_v13_cuda_matrix = list(itertools.product(
+    (CP38, CP39, CP310, CP311), (LINUX, WINDOWS)))
+_v13_aarch64_matrix = list(itertools.product(
+    (CP38, CP39, CP310, CP311), (LINUX_AARCH64,)))
+_v13_rocm_matrix = list(itertools.product(
+    (CP38, CP39, CP310, CP311), (LINUX,)))
```
Comment (asi1024) [inline check_release_assets.py:30]: Python 3.8 support will be dropped in CuPy v13.
```suggestion
_v13_cuda_matrix = list(itertools.product(
    (CP39, CP310, CP311), (LINUX, WINDOWS)))
_v13_aarch64_matrix = list(itertools.product(
    (CP39, CP310, CP311), (LINUX_AARCH64,)))
_v13_rocm_matrix = list(itertools.product(
    (CP39, CP310, CP311), (LINUX,)))
```

**Comment by takagi** [comment]
I'll make v13.0.0a1 release of cupy-release-tools after this merge.

**Review by asi1024 — APPROVED** [review]

**Pull request merged**